app/lib/features/cross_signing/cross_signing.dart

```diff
@@ -735,28 +735,38 @@ class CrossSigning {
       return;
     }
     _processMap[flowId]?.stage = 'm.key.verification.key';
-    event.getVerificationEmoji().then((emoji) {
-      showModalBottomSheet(
-        context: rootNavKey.currentContext!,
-        builder: (BuildContext context) => Container(
-          decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(15),
-          ),
-          child: _buildOnKey(context, event, flowId, emoji),
+    // event.getVerificationEmoji().then((emoji) {
+    //   showModalBottomSheet(
+    //     context: rootNavKey.currentContext!,
+    //     builder: (BuildContext context) => Container(
+    //       decoration: BoxDecoration(
+    //         borderRadius: BorderRadius.circular(15),
+    //       ),
+    //       child: _buildOnKey(context, event, flowId, emoji),
+    //     ),
+    //     isDismissible: false,
+    //   );
+    // });
```

Comment:
Let's just remove code we don't need anymore. We can always look it up in the git history, if needed.
```suggestion
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/883/commits/e89fe6f1714890e4363daf7b8b613663b7cb0d42